controllers/snapshotenvironmentbinding_controller.go

```diff
@@ -205,7 +206,16 @@ func (f *linkedRemoteSecretsFinalizer) Finalize(ctx context.Context, obj client.
 		return finalizer.Result{}, nil
 	}
 
-	target := rapi.RemoteSecretTarget{Namespace: snapshotEnvBinding.Namespace}
+	// Get the Environment CR
```

Comment:
Is this change tested? I assume it is because of the possibility of the remote-cluster targets?

Reply:
yep